rosetta/rosetta/projects/pax/xla_flags/gpt-126m.env

```diff
@@ -0,0 +1,12 @@
+set -x
+export XLA_FLAGS="\
+    --xla_gpu_enable_latency_hiding_scheduler=true \
+    --xla_allow_excess_precision \
```

Comment:
Its color is different with respect to other values. Could you please check whitespaces around?